### DISCUSSION TITLE
Southwest Companion Pass

### DISCUSSION THREAD

**Discussion by pizzamark** [jdholtz/auto-southwest-check-in#253] [Q&A] [answered]
Per the current FAQ, it states, "To ensure your companion is also checked in, you can set up their reservation or account separately in the configuration file."

Curious if I was to execute the script w/the account option (vs conf #, first name, last name) if it would pick up the companion pass this way, or if the only reliable method is via the 'reservations' portion in the config.json?

**Comment by jdholtz** [comment, marked as answer]
You can put either the reservation of your companion (the companion’s name and confirmation number) or the companion’s account (username and password) in the config file for a companion to be checked in as well. In short, either way works. 

I’ll adjust the question to say “you can add their reservation…” to make it a bit clearer.

**Reply by jdholtz** [reply]
Hey @pizzamark I adjusted the question's wording to make it more clear. Does my reply above answer your question?

**Reply by pizzamark** [reply]
Yes, thanks a lot for the clarification!
